go/chat/inboxsource.go

```diff
@@ -869,7 +872,7 @@ func (s *localizerPipeline) localizeConversation(ctx context.Context, uid gregor
 		conversationLocal.MaxMessages = msgs
 	} else {
 		conversationLocal.MaxMessages, err = s.G().ConvSource.GetMessagesWithRemotes(ctx,
```

Comment:
Could we just call `GetMessages` here? We could probably just delete `GetMessagesWithRemotes` after this change.